src/main/resources/public/js/components/catalog-card.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import '../../css/components/catalog-card.css';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { i18n } from '../config/i18n-config';
+import { t } from '@lingui/macro';
+import LabelSection from './label-section';
+
+class CatalogCard extends React.PureComponent {
+    openInstallAppPage = () => {
+        const { app, config } = this.props;
+        const installAppPage = `/${config.language}/store/${app.type}/${app.id}`;
+        this.props.history.push({
+            pathname: installAppPage,
+            state: { app: app, config: config },
+        });
+    };
+
+    getTags = app => {
+        let buttons = [];
```

Comment:
Those aren't really buttons per se. We should rename it `tags`. What do you think ?

Reply:
Indeed, especially that we render a Tag component right after.
Renamed to tag